roles/edpm_neutron_metadata/molecule/default/verify.yml

```diff
@@ -15,3 +15,17 @@
             that:
               - metadata_config.stat.exists
             fail_msg: "metadata agent config file does not exist"
+
+    - name: Ensure that haproxy wrapper script was created
```

Comment:
haven't you switched to testinfra? Should you not kill this script as you did for the other role?